package/batocera/core/batocera-configgen/configgen/configgen/generators/scummvm/scummvmGenerator.py

```diff
@@ -46,11 +46,15 @@ def generate(self, system, rom, playersControllers, metadata, guns, wheels, game
           # rom is a directory: must contains a <game name>.scummvm file
           romPath = rom
           romName = next(rom.glob("*.scummvm")).stem
-        else:
-          # rom is a file: split in directory and file name
+        elif rom.stat().st_size < 3::
```

Comment:
Why check the file size? Can't you do this?

```python
else:
    romName = Path(rom).read_text().strip()

    if not romName:
        romName = rom.stem

    romPath = rom.parent
```

Also, shouldn't this be applied to the case where the rom is a directory and it contains a `.scummvm` file?

Reply:
Hi and hello, will be fixed soon ...
Better use `if rom.is_file():` then is this context, no?
But this method is also used in the libretro-generator ... I'll change this after testing